app/api/chemotion/search_api.rb

```diff
@@ -143,7 +132,9 @@ def advanced_search(c_id = @c_id, dl = @dl)
                                        .where(query_cond)
                                        .joins(conditions[:joins].join(' '))
         scope = order_by_molecule(scope) if conditions[:model_name] == Sample
-        scope = scope.group("#{conditions[:model_name].table_name}.id") if %w[ResearchPlan Wellplate].include?(conditions[:model_name].to_s)
+        if %w[ResearchPlan Wellplate].include?(conditions[:model_name].to_s)
```

Comment:
Ich find die Schreibweise andersrum besser lesbar: `conditions[:model_name].to_s.in?(%w[ResearchPlan Wellplate])` 

Is aber nur persönliche Vorliebe, musst du also nicht anpassen